stripe/src/main/java/com/stripe/android/paymentsheet/DefaultGooglePayRepository.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.paymentsheet
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+
+internal class DefaultGooglePayRepository() : GooglePayRepository {
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private val isReadyState = MutableStateFlow(null)
+
+    override fun isReady(): Flow<Boolean?> = isReadyState
```

Comment:
i wonder if there's any way we don't have to make this nullable.
it would be nice if `MutableStateFlow` had a similar constructor like `MutableLiveData` that took no parameter / value, and only subsequent values would be emitted.

Reply:
Good point, I can look into this when I do the real implementation.